ansible/roles/data-products-deploy/templates/model-config.json.j2

```diff
@@ -3207,74 +3207,115 @@
-          "status": ["Live", "Review", "Draft"]
-        },
-        "sort_by": {
-          "createdOn": "desc"
-        },
-        "limit": 10000
-      }
-    },
-    "tenantConfig": {
-      "tenantId": "",
-      "slugName": ""
-    },
-    "store": "azure",
-    "format": "csv",
-    "key": "druid-reports/",
-    "filePath": "druid-reports/",
-    "container": "{{ bucket }}",
-    "folderPrefix": ["slug", "reportName"]
-  },
-  "output": [{
-    "to": "console",
-    "params": {
-      "printEvent": false
-    }
-  }],
-  "parallelization": 8,
-  "appName": "ETB Metrics Model",
-  "deviceMapping": false
-  },
+	"search": {
+		"type": "none"
+	},
+	"model": "org.ekstep.analytics.model.report.ETBMetricsJob",
+	"modelParams": {
+		"reportConfig": {
+			"id": "etb_metrics",
+			"metrics": [],
+			"labels": {
+				"date": "Date",
+				"identifier": "TextBook ID",
+				"name": "TextBook Name",
+				"medium": "Medium",
+				"gradeLevel": "Grade",
+				"subject": "Subject",
+				"createdOn": "Created On",
+				"lastUpdatedOn": "Last Updated On",
+				"totalQRCodes": "Total number of QR codes",
+				"contentLinkedQR": "Number of QR codes with atleast 1 linked content",
+				"withoutContentQR": "Number of QR codes with no linked content",
+				"withoutContentT1": "Term 1 QR Codes with no linked content",
+				"withoutContentT2": "Term 2 QR Codes with no linked content",
+				"status": "Status",
+				"totalContentLinked": "Total content linked",
+				"totalQRLinked": "Total QR codes linked to content",
+				"totalQRNotLinked": "Total number of QR codes with no linked content",
+				"leafNodesCount": "Total number of leaf nodes",
+				"leafNodeUnlinked": "Number of leaf nodes with no content",
+				"l1Name": "Level 1 Name",
+				"l2Name": "Level 2 Name",
+				"l3Name": "Level 3 Name",
+				"l4Name": "Level 4 Name",
+				"l5Name": "Level 5 Name",
+				"dialcode": "QR Code",
+				"sum(scans)": "Total Scans",
+				"noOfContent": "Number of contents",
+				"nodeType": "Type of Node",
+				"term": "Term"
+			},
+			"output": [{
+				"type": "csv",
+				"dims": ["identifier", "channel", "name"],
+				"fileParameters": ["id", "dims"]
+			}],
+			"mergeConfig": {
+				"frequency": "WEEK",
+				"basePath": {{ spark_output_temp_dir }},
+				"rollup": 0,
+				"reportPath": "dialcode_counts.csv"
+			}
+		},
+		"dialcodeReportConfig": {
+				"id": "etb_metrics",
+				"metrics": [],
+				"labels": {},
+				"output": [{
+					"type": "csv",
+					"dims": ["identifier", "channel", "name"],
+					"fileParameters": ["id", "dims"]
+				}],
+				"mergeConfig": {
+					"frequency": "WEEK",
+					"basePath": {{ spark_output_temp_dir }},
+					"rollup": 1,
+					"reportPath": "dialcode_counts.csv",
+					"rollupAge": "ACADEMIC_YEAR",
+					"rollupCol": "Date",
+					"rollupRange": 1
+				}
+		},
+		"etbFileConfig": {
+			"bucket": "reports",
+			"file": "dialcode_scans/dialcode_counts.csv",
+			"filePath": "src/test/resources/reports/dialcode_counts.csv"
```

Comment:
filePath cannot have "src/test/resources/" - Change it

Reply:
Done.